### DISCUSSION TITLE
Prevent forked .github repo from being presented as my own?

### DISCUSSION THREAD

**Discussion by kingthorin** [community/community#22768] [New to GitHub]
<p>I’ve forked a <code>.github</code> repo from an org. However, it’s sponsorship info etc then show on my personal repos. I’m also now unable to create my own <code>.github</code> repo via github.new (or however you want to create a new repo).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/df5602a1f4fdb5cf726564039c8c16f11c78450a.png" data-download-href="/uploads/short-url/vRIKD2SomdMJlaUUyPsQOxBnRGi.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7570458/181073080-101abc4b-4414-401d-b8e2-791c99f75b57.png" alt="image" data-base62-sha1="vRIKD2SomdMJlaUUyPsQOxBnRGi" width="690" height="442" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/df5602a1f4fdb5cf726564039c8c16f11c78450a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">780×500 18.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is there a work around?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hello <a class="mention" href="/kingthorin">@kingthorin</a> :wave:</p>
<p>Thanks for your question!<br>
When you create a fork of an existing repository , you get a copy of that project with the intention of submitting changes to the original project.<br>
A fork may not be the appropriate solution for what you are doing here.<br>
<aside class="quote quote-modified" data-post="1" data-topic="10189">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/the-difference-between-forking-and-cloning-a-repository/10189">The difference between forking and cloning a repository</a> <a class="badge-wrapper  bullet" href="/c/github-original-series/support-protips/29"><span class="badge-category-parent-bg" style="background-color: #808281;"></span><span class="badge-category-bg" style="background-color: #BF1E2E;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Written by our Support experts, this series brings you tips and tricks to help you improve your GitHub workflows and experience.">Support Protips</span></a>
  </div>
  <blockquote>
    The difference between cloning and forking a repository on GitHub 
Just a couple weeks ago, we shared some information with you about <a href="https://github.community/t/what-the-fork/10187">forking a repository and the cool things you can do with this functionality</a>. Now you might wonder - what is the difference between forking a repository and just cloning it? 
Well, ruminate on the mysteries of these features no more! In this article we’ll go over the differences between the two methods so that you can determine the best option to use when working o…
  </blockquote>
</aside>
<br>
You will not be able to make a repository with the same name. One will need to be changed.</p>

**Comment by kingthorin** [comment]
<aside class="quote group-github-employees" data-username="ernest-phillips" data-post="2" data-topic="131234">
<div class="title">
<div class="quote-controls"></div>
 ernest-phillips:</div>
<blockquote>
<p>You will not be able to make a repository with the same name. One will need to be changed.</p>
</blockquote>
</aside>
<p>So this seems to be a miss on the functional considerations side? It seems totally reasonable for individuals to want/need their own .github repo while also having contribution requirements to other orgs or projects for which they might need to fork another .github repo.</p>

**Comment by ernest-phillips** [comment]
<p>You can absolutely make a change to the name of your fork and still create pull requests to push upstream to the original repository. This way you can easily differentiate the two.</p>

**Comment by kingthorin** [comment]
<p>Cool, never thought of trying to rename a fork. Thanks!</p>

**Comment by john-blades** [comment]
@ernest-phillips Hello, reading through this thread and have a question - if the name of the fork is changed can the forked repo owner change the access settings?  We have someone who has a forked repo and is unable to change the visibility from public to private.

**Reply by kingthorin** [reply]
Why would you need to take someone else's public work and make it private?

That seems very anti-Open Source.
